go/billing/tasks.py

```diff
@@ -166,8 +174,10 @@ def get_channel_type(transaction, tagpools):
 def get_message_description(transaction):
     if transaction['message_direction'] == MessageCost.DIRECTION_INBOUND:
         return 'Messages received'
-    else:
+    elif transaction['message_direction'] == MessageCost.DIRECTION_OUTBOUND:
         return 'Messages sent'
+    else:
+        return None
```

Comment:
Could you explain to me why this is needed?  When is a message neither outbound nor inbound?